root/edit/details/RemoveMedium.js

```diff
@@ -39,21 +39,22 @@ const RemoveMedium = ({edit}: Props): React.Element<'table'> => {
         </td>
       </tr>
 
-      {display.tracks ? (
-        <tr>
-          <th>{addColon(l('Tracklist'))}</th>
-          <td>
-            <table className="tbl">
-              <tbody>
+      <tr>
+        <th>{l('Tracklist:')}</th>
```

Comment:
How come the switch from `addColon(l('Tracklist'))`?

Reply:
Because I followed AddMedium. Will change both to use addColon.